Cogs/StudentCommands.py

```diff
@@ -144,3 +144,20 @@ async def ping(self, ctx):
         latency = round(self.bot.latency * 1000)
         await ctx.send(f'{latency} ms')
         await log(self.bot, f'{ctx.author} pinged from #{ctx.channel}, response took {latency} ms')
+
+    @commands.command()
+    async def attendance(self, ctx):
+
+        # Gets users in the same voice chat as the requester and lists their @'s.
+        try:
+            channel = ctx.message.author.voice.channel
+            members = channel.members
+            members.remove(ctx.author)
+            if members:
+                attendees = "\n".join([member.mention for member in members])
+                await ctx.message.channel.send(f"Attendees of {channel.name} required by {ctx.message.author.mention}:\n {attendees}")
+            else:
+                await ctx.message.channel.send("There are no users in your voice channel.")
+        except:
```

Comment:
Cam you make this except handle the particular exception that gets raised if the requester isn't in a vc? There's an on_command_error handle on bot.py that would get clobbered by this all-catching except

Reply:
Sure! Will do.